src/ResourceManager/ContainerRegistry/Commands.ContainerRegistry/Models/PSContainerRegistry.cs

```diff
@@ -47,20 +47,9 @@ public PSContainerRegistry(Registry registry)
         public string SkuTier { get; set; }
         public string LoginServer { get; set; }
         public DateTime? CreationDate { get; set; }
-        public ProvisioningState? ProvisioningState { get; set; }
+        public string ProvisioningState { get; set; }
```

Comment:
@Jinhuafei why was this type changed?

Reply:
SDK changes it. @djyou, do you know why we have this change in SDK?

Reply:
We used model enum as string in the latest version to reduce future breaking changes.